src/Adapter/ObjectModelHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License version 3.0
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
+ */
+
+namespace PrestaShop\Module\LinkList\Adapter;
```

Comment:
```suggestion
 */

declare(strict_types=1);

namespace PrestaShop\Module\LinkList\Adapter;
```

Reply:
Hello, I think this is out of scope. This is a merge release who aims to merge `dev` into `master` 😉 . If you want to improve the code on `dev` I think it's better to submit another Pull Request.

- It will keep the scope of the PRs clear
- This PR will be mentioned in the ChangeLog, helping users to understand what changed
- If (let's say we're out of luck) the changes you submit change behavior it will be easier for QA to verify them, without altering the QA scope of this PR which is very large as this is a Release PR